<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(125, 203, 255);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 178, 241);
border-radius: 100px;</string>
   </property>
   <widget class="QWidget" name="filling1" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="filling2" native="true">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>0</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>50</y>
      <width>350</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {

font: 90pt &quot;Arial&quot;;
color:white;
background-color: rgb(255, 178, 241);
font: 90pt &quot;Kristen ITC&quot;;
}
</string>
    </property>
    <property name="text">
     <string>Kasta</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getStartedButton">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>389</y>
      <width>450</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(125, 203, 255);
color: white;
border-radius: 30px;
	font: 50pt &quot;Arial&quot;;
font: 45pt &quot;Kristen ITC&quot;;
}
QPushButton:hover {
background-color: rgb(48, 193, 255);
}</string>
    </property>
    <property name="text">
     <string>Get started</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>270</y>
      <width>281</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/virtual-assistant-icon-11.jpg</normaloff>icons/virtual-assistant-icon-11.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>400</width>
      <height>400</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>150</y>
      <width>710</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font: 50pt &quot;Arial&quot;;
color:white;
background-color: rgb(255, 178, 241);
font: 50pt &quot;Kristen ITC&quot;;
}
</string>
    </property>
    <property name="text">
     <string>Your virtual assistant</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font: 75 italic 18pt &quot;Arial&quot;;
font: 75 18pt &quot;Kristen ITC&quot;;
</string>
    </property>
    <property name="plainText">
     <string>- weather information,
- browser searching,
- system monitoring, 
- time information,
- processes automation,
- note-taking,
- notification</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>580</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/facebookicon.png</normaloff>icons/facebookicon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>580</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/instaicon.png</normaloff>icons/instaicon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>580</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/githubicon.png</normaloff>icons/githubicon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="registerButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>490</y>
      <width>450</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:rgb(125, 203, 255);
color: white;
border-radius: 30px;
	font: 50pt &quot;Arial&quot;;
font: 45pt &quot;Kristen ITC&quot;;
}
QPushButton:hover {
background-color: rgb(48, 193, 255);
}</string>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <zorder>filling1</zorder>
   <zorder>filling2</zorder>
   <zorder>label</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_2</zorder>
   <zorder>plainTextEdit</zorder>
   <zorder>getStartedButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>registerButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
